### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by mgwalker**
### This pull request changes...

- updates the entire dependency tree for both frontend and backend to latest in-range versions
  - I deleted the `package-lock.json` in both projects, deleted their `node_modules` directories, and then re-ran `npm install` to get the latest versions of whole trees. Otherwise `npm` doesn't really update downstream dependencies. This got everything to its newest state and updated the lockfiles accordingly.
  - This resulted in an update to the design system that should affect us, but it caused some of our test snapshots to change a little bit. This PR updates those snapshots.
  - The linter got updated too and it found some issues with paths that previous versions of the linter didn't find, so this fixes those too.

### This pull request is ready to merge when...

- [x] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author
- N/A ~The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented~
- N/A ~The change has been documented~
- [x] Changelog is updated as appropriate

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,29 +1,15 @@
...
-See our [release history](https://github.com/18F/cms-hitech-apd/releases)
-
-[#2081]: https://github.com/18F/cms-hitech-apd/pull/2081
-[#2086]: https://github.com/18F/cms-hitech-apd/pull/2086
-[#2087]: https://github.com/18F/cms-hitech-apd/pull/2087
-[#2090]: https://github.com/18F/cms-hitech-apd/pull/2090
-[#2088]: https://github.com/18F/cms-hitech-apd/pull/2088
-[#2091]: https://github.com/18F/cms-hitech-apd/pull/2091
\ No newline at end of file
+See our [release history](https://github.com/18F/cms-hitech-apd/releases)
```
Comment (mgwalker) [inline CHANGELOG.md]: Resetting the changelog for the next release.

File: api/routes/auth/roles/post.endpoint.js [hunk 2]
```diff
@@ -75,10 +75,11 @@ describe('auth roles endpoint | POST /auth/roles', () => {
...
 
-      const activities = await db('auth_role_activity_mapping')
-        .where({ role_id: role.id })
-        .select('activity_id')
-        .map(activity => activity.activity_id);
+      const activities = (
+        await db('auth_role_activity_mapping')
+          .where({ role_id: role.id })
+          .select('activity_id')
+      ).map(activity => activity.activity_id);
```
Comment (mgwalker) [inline api/routes/auth/roles/post.endpoint.js]: I'm kind of surprised this was working in the first place. 🤷‍♂

**Comment by pphillips-fearless** [comment]
there is a lot of code commented out in web/src/stories/index.js, should it be deleted?

**Comment by mgwalker** [comment]
Yes, I think we should delete all of the storybook stuff. We haven't been maintaining it and it's terribly out of date, but I figured that was a lot of unrelated stuff to put into this PR. So I just commented out the one file that still referenced the storybook dependency.

**Pull request merged**